static/index-web.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <script src="/ui.js" async></script>
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <link rel="stylesheet" href="/font/font.css" />
```

Comment:
Add the `<link rel="preload"...>` for the font files themselves.

Reply:
And then if you're doing that, the font CSS doesn't need to be loaded independently, right?

Reply:
I think the font css still needs to be loaded because it setups up the `@font-face` properties. But it should be fine with just the `preload` value

> his example is trivial, as the browser probably discovers the <link rel="stylesheet"> and <script> elements in the same chunk of HTML as the preloads, but the benefits can be seen much more clearly the later resources are discovered and the larger they are. For example:
>
> Resources that are pointed to from inside CSS, like fonts or images.
https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content